export const user =async(req,res)=>{
    try {
        
    } catch (error) {
        
    }
}